commands/ipfailover.go

```diff
@@ -190,6 +190,7 @@ func RunIpFailoverList(c *core.CommandConfig) error {
 }
 
 func RunIpFailoverAdd(c *core.CommandConfig) error {
+	c.Printer.Infof("Adding an IP Failover group to a LAN...")
```

Comment:
I think it makes sense to also add the datacenter id, lan id info. This applies also to line 222 from below